docs/pages/versions/v39.0.0/sdk/map-view.md

```diff
@@ -63,23 +63,32 @@ If you have already integrated Google Sign In into your standalone app, this is
+- **If you have not configured Google Sign In**
+  1.  Build your app, take note of your Android package name (eg: in _app.json_, the value of `android.package` might be `"ca.brentvatne.growlerprowler"`)
   2.  Open your browser to the [Google API Manager](https://console.developers.google.com/apis) and create a project.
-  3.  Once it's created, go to the project and enable the **Google Maps SDK for Android**
-  4.  Go back to <https://console.developers.google.com/apis/credentials> and click **Create Credentials**, then **API Key**.
+  3.  Once it's created, go to the project and enable the **Maps SDK for Android**
+  4.  Go to <https://console.developers.google.com/apis/credentials> and click **Create Credentials**, then **API Key**.
   5.  In the modal that popped up, click **RESTRICT KEY**.
-  6.  Choose the **Android apps** radio button under **Key restriction**.
+  6.  Under **Key restrictions** -> **Application restrictions**, choose the **Android apps** radio button.
   7.  Click the **+ Add package name and fingerprint** button.
-  8.  Add your `android.package` from `app.json` (eg: `ca.brentvatne.growlerprowler`) to the Package name field.
+  8.  Add your `android.package` value from step #1 (eg: `ca.brentvatne.growlerprowler`) to the Package name field.
   9.  Run `expo fetch:android:hashes`.
-  10. Copy `Google Certificate Fingerprint` from the output from step 9 and insert it in the "SHA-1 certificate fingerprint" field.
-  11. Copy the API key (the first text input on the page) into `app.json` under the `android.config.googleMaps.apiKey` field. [See an example diff](https://github.com/brentvatne/growler-prowler/commit/3496e69b14adb21eb2025ef9e0719c2edbef2aa2).
+  10. Copy `Google Certificate Fingerprint` from the output from step 9 and insert it in the "SHA-1 certificate fingerprint" field and click **Done**.
+  11. Copy the **API Key** value from the Credentials page into your `app.json` under the `android.config.googleMaps.apiKey` field. [See an example diff](https://github.com/brentvatne/growler-prowler/commit/3496e69b14adb21eb2025ef9e0719c2edbef2aa2).
   12. Press `Save` and then rebuild the app like in step 1.
 
 #### Deploying to the Google Play Store
 
-Since your app is most likely using App Signing by Google Play, you will need to grab their app signing certificate in production rather than the upload certificate returned by `expo fetch:android:hashes`. You can do this by grabbing the signature from Play Console -> Your App -> Release management -> App signing, and then going to the [API Dashboard](https://console.developers.google.com/apis/) -> Credentials and adding the signature to your existing credential.
+Since your app is most likely using App Signing by Google Play, you will need to grab their app signing certificate in production rather than the upload certificate returned by `expo fetch:android:hashes`.  You do this by:
```

Comment:
```suggestion
Since your app is most likely using App Signing by Google Play, you will need to grab their app signing certificate in production rather than the upload certificate returned by `expo fetch:android:hashes`. To do this:
```